File: 3392-count-subarrays-of-length-three-with-a-condition/3392-count-subarrays-of-length-three-with-a-condition.py
class Solution:
    def countSubarrays(self, nums: List[int]) -> int:
        count=0
        for i in range (len(nums)-2):
            f=nums[i]
            s=nums[i+1]
            t=nums[i+2]
            if f+t==s/2.0:
                count+=1
        return count
        

File: 3392-count-subarrays-of-length-three-with-a-condition/3392-count-subarrays-of-length-three-with-a-condition.java
class Solution {
    public int countSubarrays(int[] nums) {
        int count=0;
        for(int i=0;i<=nums.length-3;i++){
            int f=nums[i];
            int s=nums[i+1];
            int t=nums[i+2];
            if(f+t==s/2.0){
                count++;
            }
        }
        return count;
    }
}